.github/workflows/ocaml-general.yml

```
@@ -35,28 +35,21 @@ jobs:
           - amd64
           # - arm64 # it's taking too long for the purpose of validation
         node-version:
-          - iron # v18 Active LTS
+          - jod # v22 Active LTS
         os:
-          - ubuntu.22.04
+          - ubuntu.24.04
```

`build-deploy-ocaml-general` では `22.04` も指定している一方で、 `validate-ocaml-general` では `24.04` だけとなっており、 `22.04` も追加しておくのが良さそうに見えます。

https://github.com/kxcinc/pubinfra.dockerfiles/blob/ed057caa25879d4e4e7bb58b99c4c5d690ec0e90/.github/workflows/ocaml-general.yml#L107-L109

```suggestion
          - ubuntu.22.04
          - ubuntu.24.04
```

---

22.04 と 24.04 での違いは大きくない(+ PR の時点で build-deploy-ocaml-general で全パターン確認しています)ですが、 matrix になっていますのでその分追加のリソース消費が大きい。なので skip したいと思います :pray: